vendor/bat-native-ledger/src/bat/ledger/internal/wallet/create.cc

```diff
@@ -42,6 +61,30 @@ void Create::Start(ledger::CreateWalletCallback callback) {
       ledger::URL_METHOD::GET, on_req);
 }
 
+void Create::StartSafetyNetCallback(
+    int response_status_code,
+    const std::string& response,
+    const std::map<std::string, std::string>& headers,
+    ledger::CreateWalletCallback callback) {
+
+  ledger_->LogResponse(__func__, response_status_code, response, headers);
+
+  unsigned int statusCode;
```

Comment:
Please use `snake_case` and not `camelCase`

Reply:
https://google.github.io/styleguide/cppguide.html#Variable_Names

Reply:
Linked guide quotes `std::string tableName;   // Bad - mixed case.`

Reply:
A lot of original ledger code uses camelcase, but we should not be adding any new code with it. Camelcase is only for class and method names

Reply:
NONCRITICAL
Covered by: brave/brave-browser#6235 (19)